packages/node_modules/@webex/plugin-meetings/src/webex.js

```diff
@@ -1,36 +0,0 @@
-import '@webex/plugin-authorization';
```

Comment:
What are these files used for? Why are we removing them in this PR?

Reply:
The two files below was not being used anywhere and is dead code. This was discovered after troubleshooting why webrtc-adapter was not loading with @arun3528 and @taymoork2. Removing to avoid confusion, because I initially thought it was being used.

@webex/plugin-meetings/src/webex.js
@webex/plugin-meetings/webex.js

Reply:
This file looks like not being used anywhere